xcpmd/src/vm-actions-module.c

```diff
@@ -707,9 +767,11 @@ void shutdown_vpnvm_dependencies_of_vm_by_name (struct arg_node * args) {
 void shutdown_vpnvm_dependencies_of_vm_by_uuid (struct arg_node * args) {
```

Comment:
I'd like to, but metaprogramming in C is pretty gross, and I was trying to avoid hiding large volumes of code in macros. If you know of a tidy way to reduce this, I'm open to suggestions.